lib/fog/vsphere/models/compute/cdroms.rb

```diff
@@ -0,0 +1,43 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Cdroms < Fog::Collection
+        autoload :Cdrom, File.expand_path('../cdrom', __FILE__)
```

Comment:
Are you sure about this? It works fine like this and the volumes model does it like this as well...